src/ip_interpolators_mod.f90

```diff
@@ -1,19 +1,33 @@
-module ip_interpolators_mod
-  ! re-export specific interpolation routines
-  ! use bilinear_interpolator_scalar_mod
-  ! use bicubic_interpolator_scalar_mod
-  ! use neighbor_interpolator_scalar_mod
-  ! use budget_interpolator_scalar_mod
-  ! use spectral_interpolator_scalar_mod
-  ! use neighbor_budget_interpolator_scalar_mod
+!> @file
+!! @brief Top-level module to export interpolation routines and constants.
+!! @author Kyle Gerheiser
 
+!> Top-level module to export interpolation routines and constants.
+!! @author Kyle Gerheiser
+module ip_interpolators_mod
+  use bilinear_interp_mod
+  use bicubic_interp_mod
+  use budget_interp_mod
+  use neighbor_interp_mod
+  use spectral_interp_mod
+  use neighbor_budget_interp_mod
   implicit none
 
+  !> @param Constant to choose BILINEAR interpolation method
```

Comment:
The param statements all go into the doxygen header at the top of the subroutine.

Reply:
It's not a subroutine. `@param` can be used to document module variables and constants.

Reply:
OK, interesting. Can we get this merged and then discuss doxygen afterwards?